experiment/simulation/main.js

```diff
@@ -0,0 +1,577 @@
+const submit_button = document.getElementById("submit");
+const next_button = document.getElementById("next-button");
+const prev_button = document.getElementById("prev-button");
+const reset_button = document.getElementById("reset-button");
+const formx1 = document.getElementById("x1");
+const formy1 = document.getElementById("y1");
+const formx2 = document.getElementById("x2");
+const formy2 = document.getElementById("y2");
+const errorx1 = document.getElementById("error-x1");
+const errorx2 = document.getElementById("error-x2");
+const errory1 = document.getElementById("error-y1");
+const errory2 = document.getElementById("error-y2");
+// get the canvas
+
+let canvas = document.getElementById("canvas");
+// get the context
+let ctx = canvas.getContext("2d");
+let chosen_color = "green";
+let possible_color = "blue";
+// define the set of coordinates depict them as 2-d vectors
+let point1 = [];
+let point2 = [];
+let decision_parameter;
+let dp = [];
+// fix the heights of the canvas
+let height = 600;
+let width = 1200;
+canvas.height = height;
+canvas.width = width;
+// determine the number of divisions based on the distance between the two points
+let divisions = 45;
+let MAXX = 88;
```

Comment:
variables should be lowercase..

Reply:
camelCase works

Reply:
They were global constants , mark the edge boundary coordinates for the canvas , that's why they have been made in upper case .

Reply:
Use const for constants everywhere then

Reply:
done